### PR TITLE
Translate the rest of `loot-core/src/client`

### PR DESCRIPTION
**Pull request by jfdoming**

### CODE REVIEW

File: packages/loot-core/src/client/query-helpers.ts [hunk 1]
```diff
@@ -234,7 +236,7 @@ class PagedQuery extends LiveQuery {
 
       const orderDesc = getPrimaryOrderBy(this.query, defaultOrderBy);
       if (orderDesc == null) {
-        throw new Error(`refetchUpToRow requires a query with orderBy`);
+        throw new Error(t('refetchUpToRow requires a query with orderBy'));
```
Comment (MatissJanis) [inline packages/loot-core/src/client/query-helpers.ts:239]: **❓ question:** ‏do we actually show this error in the UI? If not - I don't think we need a translation for it.

File: packages/loot-core/src/client/query-hooks.tsx [hunk 2]
```diff
@@ -23,7 +25,9 @@ function makeContext(queryState, opts, QueryClass) {
     useEffect(() => {
       if (query.getNumListeners() !== 0) {
         throw new Error(
-          'Query already has listeners. You cannot use the same query context `Provider` twice',
+          t(
```
Comment (MatissJanis) [inline packages/loot-core/src/client/query-hooks.tsx:28]: Same Q for these two.

File: packages/loot-core/src/client/query-helpers.ts (hunk 1 above)
Comment (jfdoming) [reply packages/loot-core/src/client/query-helpers.ts:239]: From grepping through the codebase, this is actually dead code 😅 I'll remove translations for this, but do you think it's worth deleting? I know very little about this part of the code

File: packages/loot-core/src/client/query-hooks.tsx (hunk 2 above)
Comment (jfdoming) [reply packages/loot-core/src/client/query-hooks.tsx:28]: Both of these errors are thrown within components (at least `ManageRules.tsx`). The message would just be surfaced in the generic error handler for the app, like this:
image

File: packages/loot-core/src/client/query-helpers.ts (hunk 1 above)
Comment (MatissJanis) [reply packages/loot-core/src/client/query-helpers.ts:239]: If it's not used then yea - it makes sense to remove it. :)

File: packages/loot-core/src/client/query-hooks.tsx (hunk 2 above)
Comment (MatissJanis) [reply packages/loot-core/src/client/query-hooks.tsx:28]: Ah.. in my opinion - we should keep these in English.

Comment (jfdoming) [reply packages/loot-core/src/client/query-hooks.tsx:28]: Gotcha. In that case, let's close this PR as it only contains unused translations 😅

**Pull request closed without merging**